docs/man/docker-build.1.md

```diff
@@ -14,6 +14,11 @@ docker-build - Build a new image from the source code at PATH
 [**-q**|**--quiet**[=*false*]]
 [**--rm**[=*true*]]
 [**-t**|**--tag**[=*TAG*]]
+[**-m**|**--memory**[=*MEMORY*]]
+[**--memory-swap**[=*MEMORY-SWAP]]
+[**-c**|**--cpu-shares**[=*0*]]
```

Comment:
standard markdown processing of \*\* bold \*\* and \* italics \*.  But in this case it did catch a minor issue as the MEMORY-SWAP param doesn't end with a "`*`", so it continued italics until it processed "termination \*" on the next line.